osvr/RenderKit/RenderManagerOpenGL.cpp

```diff
@@ -23,15 +23,18 @@ Sensics, Inc.
 // limitations under the License.
 
 #include "RenderManagerOpenGLVersion.h"
+#ifndef OSVR_ANDROID
 #include <SDL.h>
```

Comment:
This is not a priority at the moment. The SDL code didn't build without errors, and we don't need it for the high priority targets we're shooting for (Unreal, Unity, and custom Android apps that make their own OpenGL views in Java before even creating the render manager instance, like the one in OSVR-Android-Samples). It should be a capability check (OSVR_SDL) but we'd need to move the Android check into the cmake. Feel free to file an issue to do that, but I don't think it holds up this pull request.